### PR TITLE
Sync Network updates

### PR DESCRIPTION
**Pull request by EvgenyAgafonchikov**
## Checklist

- [x] I have read the [_Submitting Changes_](https://github.com/Azure/azure-powershell/blob/preview/CONTRIBUTING.md#submitting-changes) section of [`CONTRIBUTING.md`](https://github.com/Azure/azure-powershell/blob/preview/CONTRIBUTING.md)
- [x] The title of the PR is clear and informative
- [x] The appropriate [change log has been updated](https://github.com/Azure/azure-powershell/blob/preview/CONTRIBUTING.md#updating-the-change-log)
- [x] The PR does not introduce [breaking changes](https://github.com/Azure/azure-powershell/blob/preview/documentation/breaking-changes/breaking-changes-definition.md)
- [x] If applicable, the changes made in the PR have proper test coverage
- [x] For public API changes to cmdlets:
    - [x] the changes have gone through a [cmdlet design review](https://github.com/Azure/azure-powershell-cmdlet-review-pr)
    - [x] the cmdlet markdown files were [generated using the `platyPS` module](https://github.com/Azure/azure-powershell/blob/preview/documentation/development-docs/help-generation.md)

### CODE REVIEW

File: src/ResourceManager/Network/Commands.Network/VirtualNetworkGateway/GetAzureVpnClientIpsecParametersCommand.cs [hunk 1]
```diff
@@ -1,49 +0,0 @@
...
-// limitations under the License.
-// ----------------------------------------------------------------------------------
-
-using Microsoft.Azure.Commands.ResourceManager.Common.ArgumentCompleters;
-using System.Management.Automation;
-using MNM = Microsoft.Azure.Commands.Network.Models;
-
-namespace Microsoft.Azure.Commands.Network.VirtualNetworkGateway
-{
-    [Cmdlet(VerbsCommon.Get, "AzureRmVpnClientIpsecParameter"), OutputType(typeof(MNM.PSVpnClientIPsecParameters))]
```
Comment (maddieclayton) [inline src/ResourceManager/Network/Commands.Network/VirtualNetworkGateway/GetAzureVpnClientIpsecParametersCommand.cs]: Why is the cmdlet being removed? I still see it in the psd1 file, and we cannot allow breaking changes until the next breaking change release.

Comment (MikhailTryakhov) [reply src/ResourceManager/Network/Commands.Network/VirtualNetworkGateway/GetAzureVpnClientIpsecParametersCommand.cs]: have no idea why Evgeny pushed it to this branch, I'll close it and created new one. Please review 
https://github.com/Azure/azure-powershell/pull/6347

Comment (EvgenyAgafonchikov) [reply src/ResourceManager/Network/Commands.Network/VirtualNetworkGateway/GetAzureVpnClientIpsecParametersCommand.cs]: This is [removal PR](https://github.com/Azure/azure-rest-api-specs/pull/2975/files) to API specs so removing this methods should be expected.

Comment (MikhailTryakhov) [reply src/ResourceManager/Network/Commands.Network/VirtualNetworkGateway/GetAzureVpnClientIpsecParametersCommand.cs]: @EvgenyAgafonchikov  We have no approval from PS and no actions from contributors so no need to save this removal if it works...

**Pull request closed without merging**